File: Domain/Pacienti.cs

using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Domain
{
    public class Pacienti : AppUser
    {
        public ICollection<PacientiDoktori> Doktoret { get; set; } = new List<PacientiDoktori>();
        public ICollection<Tretmani> Tretmanet { get; set; } = new List<Tretmani>();
        public ICollection<XRay> XRays { get; set; } = new List<XRay>();
        public ICollection<Termini> Terminet { get; set; } = new List<Termini>();
    }
}

File: client-app/src/app/models/pacienti.ts
import { Doktori } from "./doktori";
import { DoktoriDto } from "./doktoriDto";
import { PacientiDoktoriDTO } from "./PacientiDoktoriDTO";
import { Termini } from "./termini";
import { Tretmani } from "./tretmani";
import { Udhezimi } from "./udhezimi";
import { User, UserFormValues } from "./user";
import { XRay } from "./xray";

export interface Pacienti extends User {

  pacientiDoktori?: PacientiDoktoriDTO[];
  tretmanet?: Tretmani[];
  xrays?: XRay[];
  doktoret?: DoktoriDto[];
  terminet?: Termini[];
  udhezimet?: Udhezimi[];

  }
  export interface PacientiFormValues extends UserFormValues{
    pacientiDoktori?: PacientiDoktoriDTO[];
  tretmanet?: Tretmani[];
  xrays?: XRay[];
  doktoret?: DoktoriDto[];
  terminet?: Termini[];
  udhezimet?: Udhezimi[];
  }